docs/rfc/rfc-19-random-beacon-v2.adoc

```diff
@@ -61,9 +61,9 @@ operators noticed the problem.
 New groups should be created with a fixed, governable frequency of relay
 requests. Instead of a ticket-based approach for group selection, we should
 use sortition pools. Group creation start transaction should be embedded into
-relay request transaction and should lock a sortition pool. From this moment,
-no operator can enter or leave the pool. Once a new relay entry appears on the
-chain, all off-chain clients should perform group selection optimistically, 
+transaction submitting relay entry and should lock a sortition pool. From this
+moment, no operator can enter or leave the pool. Once a new relay entry appears
+on the chain, all off-chain clients should perform group selection optimistically, 
```

Comment:
Negative. The pool is locked in the relay request transaction. The event to start DKG off-chain is emitted in relay entry transaction. The seed used for the sortition is unknown at the moment the pool is locked.

Reply:
Right, the frequency applies to creation of new groups. Improved the description in b0ed2dd71ecdf63263790d0a5bf7670bfb3a1a16.